### PR TITLE
3v3 Wagers now set correctly

### LINKED ISSUE #527: Tank in the first 3v3 wager spot causes the 3v3 wager to not save correctly
**Describe the bug**
When setting a 3v3 wager, if you have a tank in the first spot (at the top), then it will not save the tank team correctly. The wager3v3 saves, but not the team.

**To Reproduce**
1. Go to Setup and setup a 3v3 wager.
2. Make sure the wager has a tank in the top slot.
3. Receive a weird error from the backend.

**Expected behavior**
Should just set correctly.

**Screenshots**
image

**Additional context**
Probably just an issue with the API call.

### PR DESCRIPTION
**Pull request by blaiadam**
**Description**
3v3 Wagers now have the correct tank id array passed to them. Previously it was not passing an array that was always length 3.

**Resolves These Issues**
Resolves #527 

**Type of change**
Check options that are relevant.

- [x] Bug fix (fixes a bug issue)
- [ ] New feature (adds functionality)
- [ ] This fix or feature will cause existing functionality to not work as expected. (Please elaborate in Description.)

**Steps to Verify Functionality**
1. Go to armory.
2. Set 3v3 wager.
3. Remove 3v3 wager.
4. Make sure you db is updating correctly with the money and favorite tank team.

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code
- [x] My changes pass Flow, build without errors and (if applicable) pass Jest tests.
- [ ] This change requires a update in the design document (if applicable)

### CODE REVIEW

File: src/backend/controllers/tankController.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ exports.unfavoriteTankTeam = async (req: Request, res: Response) => {
 	}
 
 	// Find the user and set favoriteTank to null.
-	const user = await User.findById(req.user.id, 'wager money favoriteTanks');
+	const user = await User.findById(req.user.id, 'wager3v3 money favoriteTanks');
```
Comment (Derplime) [inline src/backend/controllers/tankController.js:233]: Bleh the copy/pasta likely is the culprit here

File: src/backend/controllers/tankController.js [hunk 2]
```diff
@@ -239,7 +239,7 @@ exports.unfavoriteTankTeam = async (req: Request, res: Response) => {
 			.json({ msg: 'Could not find user in db'});
 	}
 
-	const addBack = user.money + user.wager;
+	const addBack = user.money + user.wager3v3;
 	user.money = addBack;
```
Comment (Derplime) [inline src/backend/controllers/tankController.js:243]: Ditto

**Review by Derplime — APPROVED** [review]
Looks good on my end

File: src/backend/controllers/tankController.js (hunk 1 above)
Comment (blaiadam) [reply src/backend/controllers/tankController.js:233]: Yeah I made this I'm pretty sure, so mb.

**Pull request merged**